spec/generators/install/install_generator_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+require 'dummy/config/environment'
+require 'ammeter/init'
+require 'generators/draper/install_generator'
+
+describe Draper::Generators::InstallGenerator do
+  destination File.expand_path('../tmp', __FILE__)
+
+  before { prepare_destination }
+  after(:all) { FileUtils.rm_rf destination_root }
+
+  describe 'the application decorator' do
+    subject { file('app/decorators/application_decorator.rb') }
+
+    describe 'naming' do
+      before { run_generator %w(YourModels) }
```

Comment:
I don't think you need the `%w(YourModels)` here since this generator doesn't take any arguments.